tsconfig.json

```diff
@@ -0,0 +1,69 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Comment:
Gezien dit een app is die door Node.js gedraait gaat worden is het niet nodig om zo'n lage compile target op te geven. `es2018` is geloof ik 100% ondersteund in de meest recente LTS van Node.js.